crates/hyperdrive-math/src/long/close.rs

```diff
@@ -66,9 +68,16 @@ mod tests {
         for _ in 0..*FAST_FUZZ_RUNS {
             let state = rng.gen::<State>();
             let in_ = rng.gen_range(fixed!(0)..=state.effective_share_reserves());
-            let normalized_time_remaining = rng.gen_range(fixed!(0)..=fixed!(1e18));
+            let maturity_time = state.checkpoint_duration();
```

Comment:
shouldn't this be `position_duration`?

Reply:
assuming the current time is 0; otherwise it should be `now + position_duration`

Reply:
ooh yeah `position_duration` makes more sense